.github/workflows/haskell-ci.yml

```diff
@@ -6,11 +6,11 @@
 #
 #   haskell-ci regenerate
 #
-# For more information, see https://github.com/haskell-CI/haskell-ci
+# For more information, see https://github.com/andreasabel/haskell-ci
```

Comment:
>>> My fork uses the ubuntu-20.04 image rather than the ubuntu-18.04 image which allows it to use the latest version of action/checkout.
>>
>> And why do we need the latest version of action/checkout?
>
> Not needed.
> But you might get harassed by dependabot if you don't use it...

As discussed in the previous PR, unless there is a really convincing reason,  I would really be happier continuing to use the main repo. In particular, we don't use dependabot yet, so I think the point is moot.

Any chance you could revert this change?

Reply:
[I have created this workflow](https://github.com/agda/agda-stdlib/commits/master/.github/workflows/haskell-ci.yml) and I am maintaining it.  Is there a problem if I maintain it my way?

If you must, 
- please install https://github.com/haskell-CI/haskell-ci
- type `haskell-ci regenerate`
- undo the changes it does to the triggers (`on` section in the `haskell-ci.yml` file); can be easily done using git tooling like `magit` or whatever
- amend the commit

Reply:
I did that in a parallel PR, so choose whatever you like better:
- #2187